Tweaks to model definition (#252)
CDCgov/ww-inference-model: model_definition.md

Whenever the sum of the wastewater catchment population sizes $\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ is less than the total population size $n$, we use an additional subpopulation of size $n - \sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ to model individuals in the population who are not covered by wastewater sampling.
In this case, we refer to the subpopulation not covered by wastewater as the reference subpopulation, denoted by $k=0$.

The total number of subpopulations is then $K_\mathrm{total} = K_\mathrm{sites} + 1$: the subpopulation to account for individuals not covered by wastewater sampling plus the $K_\mathrm{sites}$ subpopulations with sampled wastewater.

The model without wastewater (hospital admissions only model) is therefore a special case of the model where $K_\mathrm{sites} = 0$ and $K_\mathrm{total} = 1$, with subpopulation size $n_k = n$, the total population.

> In the case where the sum of the wastewater site catchment populations meets or exceeds the total population ($\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$) the model does not assign a subpopulation without sampled wastewater. In that case, the reference subpopulation, denoted by $k=0$ is assigned to the largest wastewater catchment area, and the total number of subpopulations is equal to the number of sites, $K_\mathrm{total} = K_\mathrm{sites}$.

Reviewer: ```suggestion
If the sum of the wastewater site catchment populations meets or exceeds the total population ($\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$), the model does not create a subpopulation without sampled wastewater. In that case, the largest wastewater catchment area by population serves as the reference subpopulation (denoted by $k=0$), and the total number of subpopulations is equal to the number of sites, $K_\mathrm{total} = K_\mathrm{sites}$.
```

Whenever the sum of the wastewater catchment population sizes $\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ is less than the total population size $n$, we use an additional subpopulation of size $n - \sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k$ to model individuals in the population who are not covered by wastewater sampling.

The total number of subpopulations is then $K_\mathrm{total} = K_\mathrm{sites} + 1$: the subpopulation to account for individuals not covered by wastewater sampling plus the $K_\mathrm{sites}$ subpopulations with sampled wastewater. We refer to first subpopulation as the reference subpopulation, denoted by $k=0$.

The model without wastewater (hospital admissions only model) is therefore a special case of the model where $K_\mathrm{sites} = 0$ and $K_\mathrm{total} = 1$, with subpopulation size $n_k = n$, the total population.

> In the case where the sum of the wastewater site catchment populations meets or exceeds the total population ($\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$) the model does not use a final subpopulation without sampled wastewater. In that case, the total number of subpopulations $K_\mathrm{total} = K_\mathrm{sites}$.

Reviewer: Need to be consistent between "first" and "final" subpopulation, and state explicitly which subpopulation is the reference subpopulation when $\sum\nolimits_{k=1}^{K_\mathrm{sites}} n_k \ge n$

To obtain the number of infections per capita $I(t)$ in the total population as a whole, we sum the $K_\mathrm{total}$ subpopulation per capita infection counts $I_k(t)$ weighted by their population sizes:

```math

> I(t) = \frac{1}{\sum\nolimits_{k=0}^{K_\mathrm{total}-1} n_k} \sum_{k=0}^{K_\mathrm{total}-1} n_k I_k(t)

Author: Yes, I can add that in the subpopulation definition. Also going to add the fix that Sam caught in reviewing the manuscript in the log R(t) model
